packages/template-retail-react-app/app/pages/cart/partials/cart-secondary-button-group.jsx

```diff
@@ -101,12 +101,15 @@ const CartSecondaryButtonGroup = ({
                             />
                         </Button>
                     )}
-                    <Button variant="link" size="sm" onClick={() => onEditClick(variant)}>
-                        <FormattedMessage
-                            defaultMessage="Edit"
-                            id="cart_secondary_button_group.action.edit"
-                        />
-                    </Button>
+                    {/* Only show Edit if the product is a variant */}
```

Comment:
nitpick
```suggestion
Only show Edit button if the product is a variant and has variation attributes
```
Although I don't even know if a variation product with no variants is allowed? 🤔

Reply:
I don't think that would be allowed. We can't add variation products to cart without selecting a variant.

But I can change the comment to whatever you think is best. I don't have strong a opinion about it

Reply:
No biggie, although I wonder since we know that a variant product should always include variant attributes then is checking `variant.variationAttributes.length > 0` necessary or just a precaution? Will it mask an API issue?

Reply:
length > 0 could be left out. Right not it's not making a difference